src/graphql/admin/root/query/accounts-details-by-custom-field.ts

```diff
@@ -0,0 +1,41 @@
+import { GraphQLEnumValueConfigMap } from "graphql"
+
+import { getAccountsConfig } from "@config"
+
+import { Accounts } from "@app"
+
+import { GT } from "@graphql/index"
+import { mapError } from "@graphql/error-map"
+import GraphQLAccount from "@graphql/admin/types/object/account"
+
+const { customFields: customFieldsSchema } = getAccountsConfig()
+
+const fields: Readonly<GraphQLEnumValueConfigMap> = customFieldsSchema.reduce(
+  (acc, val) => {
+    if (!val.array) {
```

Comment:
why would you filter arrays?

Reply:
Serialization (db and grapqhl), I tried different alternatives but was taking longer than expected. Any suggestion is welcome (input is a string and db aggregation use $toString to enable partial matches)

Reply:
does not apply after remove array